sif_parser/sif_open.py

```diff
@@ -150,3 +151,77 @@ def xr_open(sif_file, ignore_corrupt=False, lazy=None):
     return xr.DataArray(data, dims=['Time', 'height', 'width'],
                         coords=coords, attrs=new_info)
 
+def spool_open(spool_dir, ignore_missing=False):
+    """
+    Read from directory binary files and metadata and return a np.array.
```

Comment:
Could you add a bit more description for this function, e.g., what spool data is / how it should be structured.